benchmark/Support/graph_bank.jl

```diff
@@ -0,0 +1,262 @@
+using Catlab, Catlab.Theories
+using Catlab.CategoricalAlgebra
+using Catlab.Graphs
+using Catlab.Graphs.GraphGenerators
+include("SSets.jl")
+
+# Graph Functions
+# add_loops
+add_loops!(g) = add_parts!(g, :E, nparts(g, :V), src=parts(g, :V), tgt=parts(g, :V))
```

Comment:
This could be upstreamed into Catlab.Graphs. I would add an argument like `add_loops!(g, vs)` to allow people to only add loops for the vertices listed in vs.

Reply:
This is now implemented. Let me know if another version/variation is desired.